sdks/java/io/redis/src/test/java/org/apache/beam/sdk/io/redis/RedisIOTest.java

```diff
@@ -186,6 +186,32 @@ public void testWriteReadUsingRpushMethod() throws Exception {
     Assert.assertEquals(value + newValue, String.join("", values));
   }
 
+  @Test
+  public void testWriteUsingHLLMethod() throws Exception {
+    String key = "key";
+
+    Jedis jedis =
+        RedisConnectionConfiguration.create(REDIS_HOST, embeddedRedis.getPort()).connect();
+
+    PCollection<KV<String, String>> write =
+        writePipeline.apply(
+            Create.of(
+                KV.of(key, "0"),
+                KV.of(key, "1"),
+                KV.of(key, "2"),
+                KV.of(key, "3"),
+                KV.of(key, "4"),
+                KV.of(key, "5")));
+
+    write.apply(
+        RedisIO.write().withEndpoint(REDIS_HOST, embeddedRedis.getPort()).withMethod(Method.PFADD));
+
+    writePipeline.run();
+
+    long count = jedis.pfcount(key);
+    Assert.assertTrue(count == 6);
```

Comment:
Nit: could be simplified to `Assert.assertEquals(6, jedis.pfcount(key));`

Reply:
Changed

Reply:
I still see the old one, not a big deal though. 
Just a final note - could you add '[BEAM-6099]' in the beginning of commit message? Thanks

Reply:
That's odd. It is showing up in the files changed
                KV.of(key, "0"),
                KV.of(key, "1"),
                KV.of(key, "2"),
                KV.of(key, "3"),
                KV.of(key, "2"),
                KV.of(key, "4"),
                KV.of(key, "0"),
                KV.of(key, "5")))

Have modified the commit message as well. Thanks